.cirrus.yml

```diff
@@ -8,6 +8,9 @@ env:
     CIRRUS_SHELL: "/bin/bash"
     # No need to go crazy, but grab enough to cover most PRs
     CIRRUS_CLONE_DEPTH: 50
+    IMG_SFX: "${CIRRUS_BUILD_ID}"  # Same for all tasks of _this_ build
+    # Imgae create/import credentials (N/B: must NOT == gcp_credentials)
```

Comment:
(for later, not now) typo: imgae. And maybe you could explain what `GAC` stands for.

Reply:
Google Application Credentials (I swore I spelled it all out somewhere...maybe it got lost in a past revision).  Anyway, it's not my term, it's what google calls the JSON file in their docs also.

Reply:
k, fixed.